projects/frontend/cicd/test_e2e_data_pipelines.sh

```diff
@@ -25,7 +25,9 @@ if [ $# -eq 0 ]
     echo "ERROR: No OAuth2 token provided."
     exit 3
 fi
-export OAUTH2_API_TOKEN=$1
+export OAUTH2_API_TOKEN="$1"
+export CYPRESS_test_environment="$2"
+export CYPRESS_grepTags="$3"
```

Comment:
I think this is not the right place to discuss why I'm adding `CYPRESS_test_environment`, because this variable exist in all @taurus projects and also in SC, and is part of the UI testing strategy to provide contextual params for long-lived assets and to avoid clash between CI/CD e2e execution and local e2e execution on different machines.

Reply:
It was used before somewhere else - however why introducing it here now? Where is it used, how, and for what purpose?